teams/wg-async.toml

```diff
@@ -42,3 +42,12 @@ name = "Async working group"
 description = "Pursuing core language and library support for async-await"
 repo = "https://github.com/rust-lang/wg-async"
 zulip-stream = "wg-async"
+
+[[zulip-groups]]
+name = "WG-async"
+
+[[zulip-groups]]
+name = "WG-async-sprint-planning"
```

Comment:
The original point of this alias was to be a subset of the members who lead a focus area or otherwise participate in sprint planning meetings. It can probably be revisited. I can follow up with a PR to make a sub-team.

Reply:
We might be able to use https://github.com/rust-lang/team/pull/833 to accomplish this.